### PR TITLE
WINUI-2632 : Modify the UG document with windows 11 style image for DataGrid

### PR DESCRIPTION
**Pull request by VasanthkumarDurairaj**
I have updated the UG for Clipboard Operations, CRUD operation, MasterDetailsView, DataVirtualization and UI Customization in DataGrid.

Task Link-https://syncfusion.atlassian.net/browse/WINUI-2632

### CODE REVIEW

File: winui/DataGrid/Master-Details-View.md [hunk 1]
```diff
@@ -459,68 +475,75 @@ For manually defined relation, the properties can be directly set to the [ViewDe
...
 
 For two levels of nesting,
 
 {% tabs %}
 {% highlight xaml %}
-<syncfusion:SfDataGrid x:Name="dataGrid"
+<dataGrid:SfDataGrid x:Name="dataGrid"
                         AutoGenerateColumns="True"
                         AutoGenerateRelations="False"
+                        v
```
Comment (susmitha-sundar) [inline winui/DataGrid/Master-Details-View.md:515]: Please remove the unwanted changes.

File: winui/DataGrid/Master-Details-View.md [hunk 2]
```diff
@@ -299,106 +306,113 @@ To define Master-Details View relations, create [GridViewDefinition](https://hel
...
 
 ## Populating Master-Details view through events
 
 You can load `ItemsSource` for [DetailsViewDataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.DetailsViewDataGrid.html) asynchronously by handling [SfDataGrid.DetailsViewExpanding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html#Syncfusion_UI_Xaml_DataGrid_SfDataGrid_DetailsViewExpanding). You can set ItemsSource in on-demand when expanding record through [GridDetailsViewExpandingEventArgs.DetailsViewItemsSource](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.GridDetailsViewExpandingEventArgs.html#Syncfusion_UI_Xaml_DataGrid_GridDetailsViewExpandingEventArgs_DetailsViewItemsSource) property in the [SfDataGrid.DetailsViewExpanding](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html#Syncfusion_UI_Xaml_DataGrid_SfDataGrid_DetailsViewExpanding) event handler. 
 
 {% tabs %}
 {% highlight xaml %}
-<syncfusion:SfDataGrid  x:Name="dataGrid"
+<dataGrid:SfDataGrid  x:Name="dataGrid"
```
Comment (susmitha-sundar) [inline winui/DataGrid/Master-Details-View.md:413]: Some of the MasterDetailsView images Header text should be change with space.

File: winui/DataGrid/Master-Details-View.md (hunk 1 above)
Comment (VasanthkumarDurairaj) [reply winui/DataGrid/Master-Details-View.md]: Removed the unwanted changes

File: winui/DataGrid/Master-Details-View.md (hunk 2 above)
Comment (VasanthkumarDurairaj) [reply winui/DataGrid/Master-Details-View.md:413]: Modified the images

**Review by AmalRajUmapathySelvam — APPROVED** [review]

**Pull request merged**